Library/Homebrew/.rubocop.yml

```diff
@@ -54,6 +54,8 @@ Metrics/BlockNesting:
 Metrics/ClassLength:
   Enabled: true
   Max: 1400
+  Exclude:
+    - 'formula.rb'
```

Comment:
I'm thinking we should probably look to address https://github.com/Homebrew/brew/issues/7984 rather than cutting out an exception for `formula.rb`.